lib/MusicBrainz/Server/Controller/Admin/WikiDoc.pm

```diff
@@ -107,11 +133,8 @@ sub delete : Local Args(0) RequireAuth(wiki_transcluder) Edit CSRFToken
 
     my $page = $c->req->params->{page};
     my $version = $c->model('WikiDocIndex')->get_page_version($page);
-    my $form = $c->form(
-        form => 'Confirm'
-    );
 
-    if ($c->form_posted_and_valid($form)) {
+    if ($c->form_posted && $c->validate_csrf_token) {
```

Comment:
It's fine to keep the old code, since we'll just have to change it back if we add a cancel button or edit note field later. (I think the `Confirm` form already supports one.) I was mainly just suggesting we didn't have to pass it to the component for now.

Reply:
Replaced both https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/3d2617080a1931162a2d4624f1603ff762ecabf2 and https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/0f153838c7a5e455af8209c1047a4bf975a18022 with https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/c57771905ad73b80d067f9ed56bd121b77fc3b95.